capabilities.json

```diff
@@ -137,11 +137,13 @@
         }
       }
     },
-    "axisOptions": {
-      "displayName": "Axes properties",
-      "displayNameKey": "Visual_AxisProperties",
+    "xAxis": {
```

Comment:
We shouldn't rename the existing properties due to fact that it breaks the existing reports.
Please keep the previous name instead.